presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -54,6 +55,32 @@ public void testCreateUnpartitioned()
         }
     }
 
+    @Test
+    public void testCreateExternalUnpartitioned()
+    {
+        String tableName = "test_basic_statistics_external_unpartitioned_presto";
+
+        onPresto().executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
+
+        try {
+            String location = getTableLocation("nation", false);
+            onPresto().executeQuery(format("" +
+                    "CREATE TABLE %s (" +
+                    "   n_nationkey bigint, " +
+                    "   n_regionkey bigint, " +
+                    "   n_name varchar(25), " +
+                    "   n_comment varchar(152)) " +
+                    "WITH (external_location = '%s')",
```

Comment:
`nation` table is `format = 'TEXTFILE'`, so use this here too

(you can verify the table format with `select count(*) from the table`)

Reply:
`textfile_field_separator = '|'` is important too